data/json/vehicle_parts.json

```diff
@@ -5104,5 +5104,81 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "count": [1, 2]}
         ]
+    },{
+        "type" : "vehicle_part",
+        "id" : "plow",
+        "name" : "Plow",
+        "symbol" : "&",
+        "color" : "blue",
+        "broken_symbol" : "*",
+        "broken_color" : "light_blue",
+        "damage_modifier" : 40,
+        "durability" : 200,
+        "item" : "sheet_metal",
+        "difficulty" : 4,
+        "location" : "under",
+        "bonus": 150,
```

Comment:
Hrmrmrm... I might have to also add a mass change to the vehicle. I know that setting that to 150,000 would make your bike a no-go, but I don't know about the other effects.